src/redux/wallets/reducer.ts

```diff
@@ -154,6 +155,23 @@ export default (state: IWalletsState = intialState, action: IAction) => {
                 }
             };
 
+        case UPDATE_TRANSACTION_STATUS:
```

Comment:
You could use `TRANSACTION_UPSERT` for this. Basically it does the same thing, but alsa adds as a new transaction is the transaction does not exists. Or do you think unwanted errorenous insert could be an issue?

Reply:
I wanted to keep separate the transaction updates that come from push notifications and the ones that are based on app calls. This case will update only the status for a specific transaction. @krisboit ?

Reply:
if the actions are doing the same thing, let's reuse them.